libres/lib/enkf/enkf_fs.cpp

```diff
@@ -228,10 +228,11 @@ enkf_fs_type *enkf_fs_alloc_empty(const char *mount_point) {
     if (util_try_lockf(fs->lock_file, S_IWUSR + S_IWGRP, &fs->lock_fd)) {
         fs->read_only = false;
     } else {
-        fprintf(stderr, " Another program has already opened filesystem "
-                        "read-write - this instance will be UNSYNCRONIZED "
-                        "read-only. Cross your fingers ....\n");
-        fs->read_only = true;
+        if (!read_only) {
+            fprintf(stderr, " Another program has already opened filesystem "
+                            "read-write - this instance will be UNSYNCRONIZED "
+                            "read-only. Cross your fingers ....\n");
```

Comment:
Perhaps change: `Cross your fingers` with something more... proffesional and descriptive?

Reply:
Is there any valid reason to actually allow this except from the runs we know is ok - like everest and webviz-ert? Should it not be allowed at all and just terminated? @sondreso ?

Reply:
running `ert gui ***.ert` now works without printing the message. BUT - and i think @DanSava  also hit this issue - during requests back and forth when running the plotter - the message `Another program has already opened filesystem read-write - this instance will be UNSYNCRONIZED read-only. Cross your fingers ....` pops up from time to time. Not sure if it is possible to fix this without major refactoring if i remember correctly?

Reply:
I think we can safely exit and don't allow users to open ERT in this scenario. Only if the `read_only` flag is set one should be allowed to continue, and these scenarios we will be responsible for (such as when using the API).

Reply:
That i don't know, but @sondreso might have some good suggestions. At least i think we should print something telling the user that this is not longer possible and ERT will terminate. But - as i stated in another comment here, the log-messages appear from time to time during a run as well in between the API-calls. Not sure where this originates - but exiting ERT at that point will kill it in the midst of a run. This needs to be investigated and we need to inject the `read-only` parameter at that point.

Reply:
@oyvindeide do you have any input to how we best exit ERT if we end up in a situation we have two different runs trying to run ERT without the read-only flag set?